app/common/templates/common/macros/move-up-down-table.html

```diff
@@ -40,3 +40,53 @@
     {% endfor %}
   </dl>
 {% endmacro %}
+
+{# knows how to diplay a summary card or flat list of components #}
+{% macro moveUpDownTable(table_rows, component=None) %}
+  {% if component.is_group %}
+    <div class="govuk-summary-card">
+      <div class="govuk-summary-card__title-wrapper">
+        <h2 class="govuk-summary-card__title">
+          {{ component.name }}
+          {% if component.conditions %}
+            {{
+              govukTag({
+                "text": "Conditional",
+                "classes": "govuk-tag--pink govuk-!-margin-left-1",
+              })
+            }}
+          {% endif %}
+        </h2>
+        <ul class="govuk-summary-card__actions">
+          <li class="govuk-summary-card__action">
+            <a class="app-link-button" href="#">Manage group</a>
+          </li>
+          <li class="govuk-summary-card__action">
+            {# todo: this is based on the template context again, there's a way of generically passing down context #}
```

Comment:
Please can we not - can we just pass in the things we need to the macro? 🙏

Reply:
yeah sorry comment phrasing bad - I meant in this instance theres a way of passing down the context we need into this macro, vs. using `with context` (which the current macro does)

Reply:
https://github.com/communitiesuk/funding-service/pull/606/commits/b76f0d5f7f91c4c5cbcbc878867c01b5762b2eed